engine/reference/commandline/scout_stream.md

```diff
@@ -0,0 +1,16 @@
+---
+datafolder: scout-cli
+datafile: docker_scout_stream
+title: docker scout stream
+---
+<!--
+This page is automatically generated from Docker's source code. If you want to
+suggest a change to the text that appears here, open a ticket or pull request
+in the source repository on GitHub:
+
+https://github.com/docker/scout-cli-plugin
```

Comment:
I guess this won't really work, as this is the private repository (currently) 🤔

Reply:
(not work as in: contributors won't be able to contribute there 😅)

Reply:
- Point to `scout-cli`
- Remove "or pull request" and only encourage tickets?

Reply:
Where's that md content coming from?

Reply:
It's manually created in the docs repository, it's needed for Jekyll to render a .html file at this location. But the content for the page gets templated from the corresponding .yaml file

Reply:
So, could we just change the text to be:

```
suggest a change to the text that appears here, open a ticket in the source repository on GitHub:
https://github.com/docker/scout-cli
```